recipes/odbc2parquet/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "odbc2parquet" %}
+{% set version = "2.0.2" %}
+{% set build_num = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pacman82/odbc2parquet/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 8d635c8407d7e79816f9c2cc102f0cff28876e2ea95f47f82b8e108c534b0846
+
+build:
+  number: {{ build_num }}
+  script:
+    - cargo install --root "$PREFIX" --path .
```

Other recipes seem to work without this arg (probably set by rust activation)
```suggestion
    - cargo install --path .
```

---

Doesn't seem to work